src/Services/Order/OrderCreateService.php

```diff
@@ -333,7 +338,7 @@ private function getOrderItems(array $data)
 					],
 					'properties'      => [
 						[
-							'typeId'    => OrderPropertyType::SELLER_ACCOUNT,
```

Comment:
might this have an impact on aftersales processes?

Reply:
I searched for `OrderPropertyType::SELLER_ACCOUNT` in the whole directory and can't find any other place where it is used

Reply:
did you also search for 10? 😄

Reply:
now yes 😄 but no results either